packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -2615,10 +2618,11 @@ export default abstract class SqlIntegration
 
     return format(
       `-- Feature Evaluation Diagnostics Query
-      SELECT * FROM (
+      WITH _featureEvalQuery AS (
```

Comment:
nit: I don't know why, we do use 2 underscores as the prefix. Can we update it here too?

`__featureEvalQuery`